modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -4117,6 +4072,13 @@
             <xsd:complexContent>
                 <xsd:extension base="baseSplitterType">
                 <xsd:attributeGroup ref="optionalExpressionAttributes"/>
+                <xsd:attribute name="filterOnErrorType" type="errorIdentifierType" use="optional">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            The full name of the error types or a comma separated list of full names, to filter out of the resulting payload when it occurs. This is useful to use validations inside this component.
```

Comment:
Improve with something like: A comma separated list error types for that should be handled by dropping the split part, such that it won't be included in the aggregated response collection, rather than propagating the error.